inference-engine/src/inference_engine/ie_core.cpp

```diff
@@ -485,6 +485,12 @@ class Core::Impl : public ICore, public std::enable_shared_from_this<ICore> {
         return details::ReadNetwork(model, weights, extensions);
     }
 
+    CNNNetwork ReadNetwork(const std::string& model, const Blob::CPtr& weights, const std::vector<IExtensionPtr>& exts) const override {
+        OV_ITT_SCOPE(FIRST_INFERENCE, itt::domains::IE_RT, "Core::Impl::ReadNetwork from memory with extensions");
+        return details::ReadNetwork(model, weights, exts);
+    }
```

Comment:
since we register CPU extensions in the ctor, do we need this custom overload then? I suppose Core already knows about CPU extensions..